ghost/admin/app/components/gh-date-time-picker.js

```diff
@@ -121,15 +121,15 @@ export default class GhDateTimePicker extends Component {
 
         // unless min/max date is at midnight moment will disable that day
         if (minDate === 'now') {
-            this.set('_minDate', moment(moment().format(DATE_FORMAT)));
+            this.set('_minDate', moment.tz(moment().format(DATE_FORMAT)), blogTimezone);
```

Comment:
Thank you for your contribution! I only have two remarks that should be easy to fix.

- `blogTimezone` has been added as an argument to `set` instead of `moment.tz`
- Say that the current date in the timezone of the server is `2022-01-02`. And your local timezone, the current date is `2022-01-01`. Then this line will run the following code:
```
this.set('_minDate', moment.tz('2022-01-01', blogTimezone));
```
Which means that the minimum selectable day is still not right, sadly. 

I think we need one extra change here (for both the minimum and maximum date):

```suggestion
            this.set('_minDate', moment.tz(moment().tz(blogTimezone).format(DATE_FORMAT), blogTimezone));
```

Reply:
You are absolutely right! Let me quickly push a new commit implementing these changes. 🚀

Reply:
One quick question about the new change. Do you think we still need the outer `moment.tz` call if we are setting the timezone of the date inside?
```suggestion
            this.set('_minDate', moment(moment().tz(blogTimezone).format(DATE_FORMAT)));
```